swagger.yaml

```diff
@@ -49,6 +49,17 @@ components:
           type: string
         type:
           type: string
+    Report:
+      type: object
+      properties:
+        from:
+          type: string
+        to:
+          type: string
+        subject:
+          type: string
+        text:
+          type: string
```

Comment:
As discussed, this does not reflect the actual browser request schema/required route body info.